lib/sql-string.js

```diff
@@ -61,6 +61,10 @@ SqlString.escape = function(val, stringifyObjects, timeZone, dialect, field) {
     return val + '';
   }
 
+  if (field && field.type && field.type.validate) {
+    field.type.validate(val);
```

Comment:
I'm talking about every type of query that can be generated by Sequelize. Since we already know the data type of the columns that we are querying we can prevent an error from being thrown by the database.